hooks/useRealm.tsx

```diff
@@ -244,6 +244,10 @@ export default function useRealm() {
     realmInfo,
     symbol,
     voteSymbol: realmInfo?.voteSymbol,
+    allowDiscussion:
```

Comment:
`useRealm` already surfaces `realmInfo`, there is no need to create a new field. I don't like the existing way that fields are constantly re-exported like this and I'm in the process of killing it.